book/10-git-internals/sections/maintenance.asc

```diff
@@ -168,27 +172,28 @@ dangling tree aea790b9a58f6cf6f2804eeac9f0abbe9631e4c9
 dangling blob 7108f7ecb345ee9d0084193f147cdad4d2998293
 ----
 
-In this case, you can see your missing commit after the string ``dangling commit''.
-You can recover it the same way, by adding a branch that points to that SHA-1.
+Sa kasong ito, makikita mo na ang nawawalang commit ay pagkatapos ng ``dangling commit''.
+Maaring mo itong maibalik sa parehong paraan, sa pamamagitan ng paggawa ng branch na nagtuturo sa SHA-1 na iyon.
 
 [[_removing_objects]]
-==== Removing Objects
+==== Pagtanggal Ng Mga Object
 
-There are a lot of great things about Git, but one feature that can cause issues is the fact that a `git clone` downloads the entire history of the project, including every version of every file.
-This is fine if the whole thing is source code, because Git is highly optimized to compress that data efficiently.
-However, if someone at any point in the history of your project added a single huge file, every clone for all time will be forced to download that large file, even if it was removed from the project in the very next commit.
-Because it's reachable from the history, it will always be there.
+Maraming mahusay na mga bagay tungkol sa Git, ngunit isang tampok na maaaring maging sanhi ng mga isyu ay ang katunayan na ang isang `git clone` ay nag-dodownload ng buong kasaysayan ng proyekto, kabilang ang bawat bersyon ng bawat file. 
+Ito ay pwede kung ang lahat ay source code, dahil ang Git ay lubos na na-optimize upang i-compress ang data nang mabuti.
+Gayunpaman, kung ang isang tao sa anumang punto sa kasaysayan ng iyong proyekto ay nagdagdag ng isang malaking file, ang bawat clone para sa lahat ng oras ay mapipilitang i-download ang malalaking file, kahit na ito ay tinanggal mula sa proyekto sa mga sumunod na commit. 
+Dahil ito ay naabot mula sa kasaysayan, palagi na yang nandiyan.
 
-This can be a huge problem when you're converting Subversion or Perforce repositories into Git.
-Because you don't download the whole history in those systems, this type of addition carries few consequences.
-If you did an import from another system or otherwise find that your repository is much larger than it should be, here is how you can find and remove large objects.
+Ito ay maaaring maging isang malaking problema kapag nagko-convert ka ng Subversion o Perforce na mga repositoryo sa Git.
+Dahil hindi mo i-download ang buong kasaysayan sa mga system na iyon, ang ganitong uri ng karagdagan ay nagdadala ng ilang mga kahihinatnan.
+Kung nagsagawa ka ng pag-import mula sa ibang sistema or anuman na mas malaki ang iyong repositoryo kaysa nararapat, 
+If you did an import from another system or otherwise find that your repository is much larger than it should be, narito kung paano mo matatagpuan at alisin ang mga malalaking object.
```

Comment:
Please remove untranslated line.
`If you did an import from another system or otherwise find that your repository is much larger than it should be,`

Reply:
done